Cargo.toml

```diff
@@ -199,7 +198,6 @@ clippy.wildcard_dependencies = "deny"
 resolver = "2"
 members = [
     "crates/*",
-    "data_model/libs/*",
     "data_model/samples/*"
```

Comment:
maybe you also want to move this somewhere?

Reply:
I'd leave it to #5225